assets/src/js/orders.js

```diff
@@ -332,10 +332,17 @@ jQuery(function($) {
 
         refunds: {
 
-            do_refund: function() {
+            do_refund: async function() {
                 dokan_seller_meta_boxes_order_items.block();
-
-                if ( window.confirm( dokan_refund.i18n_do_refund ) ) {
+                
+                const isRefund = await dokan_sweetalert( dokan_refund.i18n_do_refund, { 
+                    action : 'confirm', 
+                    showCancelButton: true, 
+                    confirmButtonColor:'#28a745',
```

Comment:
Consider making the confirmButtonColor & cancelButtonColor optional but overridable. Manage a default value.